### DISCUSSION TITLE
I need support

### DISCUSSION THREAD

**Discussion by Akul1903** [eProsima/Fast-DDS#5172] [Support]
Hello, i have installed fast dds on linux the latest version v2.14.2 which has been installed, but after installing there was no setup.bash file was formed.
and without that file we are not able to achive client server architecture.

is there any new command for latest version

**Comment by JesusPoderoso** [comment]
Did you perform the [Colcon installation](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#colcon-installation) or the [CMake installation](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html#cmake-installation)?

**Reply by Akul1903** [reply]
Thanks @JesusPoderoso 
I issue was resolved
